includes/constants/class-country-test-cards.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Class Country_Test_Cards
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Constants;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+
+/**
+ * Class handling country-specific data for WooPayments
+ */
+class Country_Test_Cards extends Base_Constant {
+	/**
+	 * Map of country codes to their test card numbers
+	 * Source: https://docs.stripe.com/testing?testing-method=card-numbers#international-cards
+	 *
+	 * @var array
+	 */
+	private static $country_test_cards = [
+		'US' => '4242 4242 4242 4242',
+		'AR' => '4000 0003 2000 0021',
+		'BR' => '4000 0007 6000 0002',
+		'CA' => '4000 0012 4000 0000',
+		'CL' => '4000 0015 2000 0001',
+		'CO' => '4000 0017 0000 0003',
+		'CR' => '4000 0018 8000 0005',
+		'EC' => '4000 0021 8000 0000',
+		'MX' => '4000 0048 4000 8001',
+		'PA' => '4000 0059 1000 0000',
+		'PY' => '4000 0060 0000 0066',
+		'PE' => '4000 0060 4000 0068',
+		'UY' => '4000 0085 8000 0003',
+		'AE' => '4000 0078 4000 0001',
+		'AT' => '4000 0004 0000 0008',
+		'BE' => '4000 0005 6000 0004',
+		'BG' => '4000 0010 0000 0000',
+		'BY' => '4000 0011 2000 0005',
+		'HR' => '4000 0019 1000 0009',
+		'CY' => '4000 0019 6000 0008',
+		'CZ' => '4000 0020 3000 0002',
+		'DK' => '4000 0020 8000 0001',
+		'EE' => '4000 0023 3000 0009',
+		'FI' => '4000 0024 6000 0001',
+		'FR' => '4000 0025 0000 0003',
+		'DE' => '4000 0027 6000 0016',
+		'GI' => '4000 0029 2000 0005',
+		'GR' => '4000 0030 0000 0030',
+		'HU' => '4000 0034 8000 0005',
+		'IE' => '4000 0037 2000 0005',
+		'IT' => '4000 0038 0000 0008',
+		'LV' => '4000 0042 8000 0005',
+		'LI' => '4000 0043 8000 0004',
+		'LT' => '4000 0044 0000 0000',
+		'LU' => '4000 0044 2000 0006',
+		'MT' => '4000 0047 0000 0007',
+		'NL' => '4000 0052 8000 0002',
+		'NO' => '4000 0057 8000 0007',
+		'PL' => '4000 0061 6000 0005',
+		'PT' => '4000 0062 0000 0007',
+		'RO' => '4000 0064 2000 0001',
+		'SA' => '4000 0068 2000 0007',
+		'SI' => '4000 0070 5000 0006',
+		'SK' => '4000 0070 3000 0001',
+		'ES' => '4000 0072 4000 0007',
+		'SE' => '4000 0075 2000 0008',
+		'CH' => '4000 0075 6000 0009',
+		'GB' => '4000 0082 6000 0000',
+		'AU' => '4000 0003 6000 0006',
+		'CN' => '4000 0015 6000 0002',
+		'HK' => '4000 0034 4000 0004',
+		'IN' => '4000 0035 6000 0008',
+		'JP' => '4000 0039 2000 0003',
+		'MY' => '4000 0045 8000 0002',
+		'NZ' => '4000 0055 4000 0008',
+		'SG' => '4000 0070 2000 0003',
+		'TW' => '4000 0015 8000 0008',
+		'TH' => '4000 0076 4000 0003',
+	];
+
+	/**
+	 * Get test card number for a specific country.
+	 *
+	 * @param string $country_code Two-letter country code.
+	 * @return string Test card number
+	 */
+	public static function get_test_card_for_country( string $country_code ) {
+		return self::$country_test_cards[ $country_code ] ?? self::$country_test_cards['US'];
```

Comment:
What do you think of using the country codes in `Country_Code` for consistency? (you'll probably need to flip the `include_once` in `includes/class-wc-payments.php`)

Reply:
Makes sense, applied in 11a914b6715f353b94d7c8a286cf1f57cd30efa5 but let me know if I misunderstood your comment and you meant something different.

Reply:
That's great, thank you! Just this one missing:
```suggestion
		return self::$country_test_cards[ $country_code ] ?? self::$country_test_cards[ Country_Code::UNITED_STATES ];
```